docs/usage.md

```diff
@@ -127,17 +127,17 @@ An example database sheet can look as follows, where 7 tools are being used, and
 `kraken2` will be run twice even though only having a single 'dedicated' database because specifying `bracken` implies first running `kraken2` on the `bracken` database, as required by `bracken`.
 
 ```csv
-tool,db_name,db_params,db_path
-malt,malt85,-id 85,/<path>/<to>/malt/testdb-malt/
-malt,malt95,-id 90,/<path>/<to>/malt/testdb-malt.tar.gz
-bracken,db1,;-r 150,/<path>/<to>/bracken/testdb-bracken.tar.gz
-kraken2,db2,--quick,/<path>/<to>/kraken2/testdb-kraken2.tar.gz
-krakenuniq,db3,,/<path>/<to>/krakenuniq/testdb-krakenuniq.tar.gz
-centrifuge,db1,,/<path>/<to>/centrifuge/minigut_cf.tar.gz
-metaphlan,db1,,/<path>/<to>/metaphlan/metaphlan_database/
-motus,db_mOTU,,/<path>/<to>/motus/motus_database/
-ganon,db1,,/<path>/<to>/ganon/test-db-ganon.tar.gz
-kmcp,db1,;-I 20,/<path>/<to>/kmcp/test-db-kmcp.tar.gz
+tool,db_name,db_params,db_type,db_path
```

Comment:
Myabe give a second example without the `db_type` column

Reply:
Added this for malt.